lib/commands/create-widget.js

```diff
@@ -38,19 +41,22 @@ module.exports = function (program) {
         util.nlog('create-widget', 'Setting up always.js for ' + widgetName + fullWidgetName + '.');
         mkdir('-p', path + '/public/js');
 
-        var jsConfig = `apos.define('${fullWidgetName}', {
-  extend: 'apostrophe-widgets',
-  construct: function(self, options) {
-    self.play = function($widget, data, options) {
+        var jsConfig = stripIndent`
```

Comment:
If we're going to make a default player going forward, then it should be a lean player. Lean players always work, even in A2, even with lean: true not set.